Frequency/Board/Map/Map.py

```diff
@@ -18,14 +18,16 @@ def __init__(self, resolution, tiles=None):
 
 
     def GenerateTiles(self):
-        maxTilesX = self.Resolution.X // 35
-        maxTilesY = self.Resolution.Y // 35
+        maxTiles = Vector2(18, 18)
+        maxTileSize = Vector2(self.Resolution.X // maxTiles.X, self.Resolution.Y // maxTiles.Y)
```

Comment:
Should it not be better when tiles always will be a squire? You can do something like:
```python
maxLength = min(self.Resolution.X // maxTiles.X, self.Resolution.Y // maxTiles.Y)
maxTileSize = Vector2(maxLength, maxLength)
```

you can also choose to change the signature of the tile object so that you only give one int